src/components/LoginForm/LoginForm.js

```diff
@@ -1,29 +1,54 @@
-import { Link } from "react-router-dom";
-import { useQuery } from "@apollo/client";
-import { USERS_QUERY } from "../../queries/queries";
-import QueryResult from "../QueryResult/QueryResult";
-import "./LoginForm.css";
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { useLazyQuery } from '@apollo/client';
+import { CREATE_SESSION_QUERY } from '../../queries/queries';
+import ConditionalLink from '../ConditionalLink/ConditionalLink';
+import './LoginForm.css';
 
 function LoginForm({ setUser }) {
-  const { loading, error, data } = useQuery(USERS_QUERY);
+  const [loginValues, setLoginValues] = useState({
+    email: '',
+    password: ''
+  });
+  const [loginUser, { loading, error, data }] = useLazyQuery(CREATE_SESSION_QUERY);
```

Comment:
I made an issue for this on our project board, but since it's that's not holding up other work that needs to be done, I'll have to get back to it later.